pkg/yurthub/otaupdate/static/static_pod.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2023 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package static
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+
+	spctrlutil "github.com/openyurtio/openyurt/pkg/controller/staticpod/util"
+	upgradeutil "github.com/openyurtio/openyurt/pkg/node-servant/static-pod-upgrade/util"
+)
+
+var (
+	DefaultManifestPath = "/etc/kubernetes/manifests"
+	DefaultUpgradePath  = "/tmp/manifests"
+)
+
+type StaticPodUpgrader struct {
+	kubernetes.Interface
+	types.NamespacedName
+}
+
+func (s *StaticPodUpgrader) Apply() error {
```

Comment:
how about use [controller.Upgrade()](https://github.com/openyurtio/openyurt/blob/master/pkg/node-servant/static-pod-upgrade/upgrade.go#L106) to complete static pod upgrade?

Reply:
Done